### PR TITLE
Campaigns API Code

### PR DESCRIPTION
**Pull request by abdullai-t**
####  Summary / Highlights

This PR introduces the implementation of models and CRUD operations, along with associated routes for Campaigns on the ME platform.

### CODE REVIEW

**Review by archx3 — COMMENTED** [review]
Great start mate.

**Review by archx3 — COMMENTED** [review]
Great job mate

File: src/apps__campaigns/models.py [hunk 1]
```diff
@@ -0,0 +1,601 @@
...
+from django.forms import model_to_dict
+
+from _main_.utils.base_model import BaseModel
+from database.models import Event, Media, UserProfile, Community, Vendor
+from database.utils.common import get_json_if_not_none, get_summary_info
+from database.utils.constants import LONG_STR_LEN, SHORT_STR_LEN
+
+# Create your models here.
+
+class CampaignAccount(BaseModel):
```
Comment (BradHN1) [inline src/apps__campaigns/models.py:11]: This was not in the spec.  Does a campaign have a single subdomain, or different subdomains per community?

**Review by Opoku-Agyemang — APPROVED** [review]
too large! please refactor and add tests!

**Pull request merged**